Pod/Classes/BONText.m

```diff
@@ -575,6 +575,15 @@ - (NSString *)debugString
     return [self debugStringIncludeImageAddresses:YES];
 }
 
+- (NSString *)debugStringLeftToRight
```

Comment:
I'm thinking that, for a debug string, the LTR character is more cruft that you don't want in your in-code debug strings, so it's probably OK to leave it as-is.